### PR TITLE
update Pantheon Professional Services doc + add Onboarding services

### PR DESCRIPTION
**Pull request by carolyn-shannon**
Closes: #5503 

## Summary

**[Onboarding Services](https://pantheon.io/docs/onboarding)** - Created new Onboarding Services page, migrating onboarding services from [Professional Services](https://pantheon.io/docs/professional-services) page. [PR 5646](https://github.com/pantheon-systems/documentation/pull/5646)

## Effect
The following changes are already committed:
- New Onboarding Services page created
- Professional Services page updated and shortened
- PS section on AGCDN updated w/current language/acronyms
- Updated category tags

## Remaining Work
- [x] OM and PS review/edits (5/20 - 👍 from both via slack)
- [x] 👍 from Allen on having two separate docs
- [x] copy edit

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Index /onboarding page w/docs search service provider
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5646 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `49` | `-7` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5646` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20079/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20079/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by EdwardAngert** [comment]
I'm working on updating and aligning some internal documentation about onboarding and PS - marking as `blocked` for now until some further discussion

**Comment by ttrowell** [comment]
@carolyn-shannon The Custom Application Services stay with the PS page.  It looks like we reference them from the Onboarding page, but then they are removed (I think) from the PS page.

**Comment by EdwardAngert** [comment]
working on this merge conflict now - it's coming from the old branch I had and then renamed

**Comment by EdwardAngert** [comment]
fixing merge conflict

File: source/content/professional-services.md [hunk 1]
```diff
@@ -25,21 +27,11 @@ The Site Migration service lowers the barrier to entry by leveraging our PS team
 
 ## Managed Updates
 
-Pantheon's [Managed Updates](/managed-updates) service keeps your site up to date with minimal interruption to end users. Allow Pantheon’s PS team to apply core and contributed plugin/module updates.
```
Comment (alexfornuto) [inline source/content/professional-services.md]: This should be updated for the link to the new page.

Comment (EdwardAngert) [reply source/content/professional-services.md]: even with the link at the end of that section? I wasn't sure about doubling it up like that

**Comment by EdwardAngert** [comment]
shared in #PS for additional feedback

File: source/content/professional-services.md (hunk 1 above)
Comment (alexfornuto) [reply source/content/professional-services.md]: :derp: - as you were.

**Pull request merged**